src/components/cylc/gscan/GScan.vue

```diff
@@ -200,38 +200,38 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 </template>
 
 <script>
-import { mapState } from 'vuex'
 import { mdiFilter } from '@mdi/js'
 import uniq from 'lodash/uniq'
-import subscriptionComponentMixin from '@/mixins/subscriptionComponent'
 import TaskState from '@/model/TaskState.model'
-import SubscriptionQuery from '@/model/SubscriptionQuery.model'
 import { WorkflowState } from '@/model/WorkflowState.model'
 import Job from '@/components/cylc/Job'
 import Tree from '@/components/cylc/tree/Tree'
 import WorkflowIcon from '@/components/cylc/gscan/WorkflowIcon'
 import { filterHierarchically } from '@/components/cylc/gscan/filters'
-import { GSCAN_DELTAS_SUBSCRIPTION } from '@/graphql/queries'
```

Comment:
This separates the component from the view code as already done for some of the other views.

* The component should be something you can mount by itself.
  * Functional dependencies should provided via `props`.
  * It shouldn't issue subscriptions or go looking in the store, it should work by itself.
  * It should be generic-ish and reusable.
  * It should be component testable!
* The "view" should contain the application logic.
  * It should start the subscription (via the required mixins).
  * It should fetch the relevant items from the store.
  * It should be functional testable.